boot-rabbitmq-thymeleaf/README.md

```
@@ -12,7 +12,16 @@ Demonstrates a message producer and consumer setup with RabbitMQ, plus a simple
 
 ---
 
-Source Code : https://sivalabs.in/2018/02/springboot-messaging-rabbitmq/
+[Source Code](https://sivalabs.in/2018/02/springboot-messaging-rabbitmq/)
+
+---
+
+## Important links
+
+* Home Page : http://localhost:8080
```

Markdownlint (0.37.0)

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)